srcpkgs/colmap/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'colmap'
+pkgname=colmap
+version=3.8
+revision=1
+build_style=cmake
+makedepends="boost-devel eigen flann-devel freeimage-devel metis-devel \
+	glog-devel gflags-devel sqlite-devel glew-devel qt5-devel cgal-devel \
+	ceres-solver-devel liblz4-devel"
+short_desc="Structure-from-Motion (SfM) and Multi-View Stereo (MVS) pipeline"
+maintainer="Adam Gausmann <adam@gaussian.dev>"
+license="GPL-3.0-or-later"
```

Comment:
not correct: https://github.com/colmap/colmap/blob/dev/COPYING.txt

Reply:
Oops, missed that field